Dockerfile.sle15

```diff
@@ -1,4 +1,4 @@
-FROM registry.suse.de/home/yeey/pintserverng/containers/pint/pint-server-lambda-function:latest
+FROM registry.suse.de/devel/pubcloud/productionservices/pint/containers/pint/pint-server-lambda-function:latest
```

Comment:
this was confusing to me at first since you cant navigate to this URL directly, but it does show up with this path at https://registry.suse.de/cgi-bin/cooverview  ... its a shame the search function on that page doesnt work right

Reply:
@guangyee missed that. This shouldn't even be here, it reveals SUSE internal information in that it shows that we run an internal registry on that server. We need to come up with a different approach.